setup.py

```
@@ -27,6 +27,7 @@ def get_version():
 	packages=find_packages(),
 	install_requires=[
 		"requests>=2.32",
+		"aiohttp>=3.8.0",
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)